src/style.css

```
@@ -1,3 +1,80 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  width: 100vw;
+  background: white;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 0 20px;
+  padding-left: 40px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px;
+}
+
+nav {
+  margin-left: auto;
+  right: 40px;
+  margin-right: 50px;
+  position: relative;
+}
+
+nav ul {
```

Use a simplify selectors everywhere
```suggestion
.nav__list {
```